app/build.gradle

```diff
@@ -117,7 +117,7 @@ dependencies {
   compile 'com.android.support:support-v4:22.2.0'
   compile 'com.squareup:otto:1.3.7'
   compile 'com.sothree.slidinguppanel:library:3.0.0'
-  compile('com.mapzen:on-the-road:0.8-SNAPSHOT') {
+  compile('com.mapzen:on-the-road:0.9-SNAPSHOT') {
```

Comment:
Current on-the-road snapshot is still 0.8-SNAPSHOT

https://github.com/mapzen/on-the-road/blob/master/gradle.properties#L3